bokeh/properties.py

```diff
@@ -544,13 +553,13 @@ def reset_changed_vars(self):
         self._changed_vars = set()
 
     def properties_with_values(self):
-        return dict([ (attr, getattr(self, attr)) for attr in self.properties() ])
+        return dict([ (attr, getattr(self, attr)) for attr in self.properties() if self.lookup(attr).serialized ])
```

Comment:
Answering to my self, no, because virtual properties should have no impact as their values are embedded in values of some other properties.